docs/src/Typeahead.doc.js

```diff
@@ -0,0 +1,147 @@
+// @flow strict
+import * as React from 'react';
+import Example from './components/Example.js';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="Typeahead"
+    description={`Use a \`Typeahead\` when you want to let the user filter a list when selecting.`}
+  />
+);
+
+card(
+  <PropTable
+    props={[
+      {
+        name: 'resultHeight',
+        type: 'string',
+        description: 'Set the height of results box',
+        href: 'basicExample',
+        defaultValue: '50vh',
```

Comment:
I think we'd want to not make it changeable? It should behaving like a normal select list where it scrolls when hitting the limits of the page instead of needing to scroll the page to see more of the dropdown.

Reply:
I am not sure if this is relevant anymore I think this should be not changeable. @simkessy